Program.cs

```diff
@@ -1,10 +1,52 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿// ---------------------------------------------------------------------------------------
+// Training ~ A training program for new joinees at Metamation, Batch- July 2023.
+// Copyright (c) Metamation India.                                                
+// ---------------------------------------------------------------------------
+// Program.cs                                                                     
+// Program to check a given number is armstrong or to print nth Armstrong number based on the user input.
+// User either provides input in the command prompt or in the appliation to get nth armstrong number or to check whether the number is armstrong respectively. 
+// ---------------------------------------------------------------------------------------
+
+namespace Training {
+   #region class Program ------------------------------------------------------------------------
+   /// <summary>Armstrong Number</summary>
+   internal class Program {
+      #region Method ----------------------------------------------
+      /// <summary>This method prints nth armstrong number when getting input from the command prompt and checks whether it is an armstrong number while executed in the file</summary>

```

Comment:
When writing comments, keep the line length shorter so you don't have to scroll horizontally